easybuild/easyblocks/g/gcc.py

```diff
@@ -831,6 +837,28 @@ def post_install_step(self, *args, **kwargs):
             else:
                 raise EasyBuildError("Can't link '%s' to non-existing location %s", target, os.path.join(bindir, src))
 
+        # Install rpath wrappers, if built with RPATH enabled
+        if self.cfg['add_rpath_wrappers']:
+            def create_rpath_wrappers(targetdir):
+                tc = get_toolchain({'name': 'GCCcore', 'version': self.version}, {})
+
+                tc.prepare_rpath_wrappers(
+                        rpath_filter_dirs=[],
+                        rpath_include_dirs=[]
+                        )
+
+                # Find location of rpath wrappers. Tempfile in prepare_rpath_wrappers seems to be
+                # in a parent directory of our current gettempdir()-location
+                wrapperpath = glob.glob(os.path.join(tempfile.gettempdir(), "tmp*", RPATH_WRAPPERS_SUBDIR))[0]
+                os.mkdir(targetdir)
+                for wrapper in os.listdir(wrapperpath):
+                    filename = os.listdir(os.path.join(wrapperpath, wrapper))[0]
+                    filepath = os.path.join(wrapperpath, wrapper, filename)
+                    move_file(filepath, os.path.join(targetdir, filename))
```

Comment:
@stderr-enst We also need to make sure that these wrappers scripts have executable permissions for all users, not just the owner (the account used to do the installation)

Reply:
This is now happening in the other PR:
https://github.com/easybuilders/easybuild-framework/blob/fc39b6609fab98de2dbbc73cb3d28f72de3fa5fb/easybuild/tools/toolchain/toolchain.py#L1045